### PR TITLE
Tackle-806:  Admin perspective for MTA 6.0

### PR DESCRIPTION
**Pull request by CarmiWisemon**
MTA 6.0 

Resolves: https://issues.redhat.com/browse/TACKLE-806

Preview: https://deploy-preview-561--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#mta-web-administration-view_ocp-42

### CODE REVIEW

File: docs/topics/mta-web-administrator-view.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+[id="mta-web-administration-view_{context}"]
+= User Interface views
+
+The {ProductName} ({ProductShortName}) {WebName} and has two user interfaces with different functionalities:
+
+* Administrator view
+* Developer view
+
+== Administrator view
+The administrator view is used by administrators to create the {ProductShortName} {WebName} instance environment.
```
Comment (rromannissen) [inline docs/topics/mta-web-administrator-view.adoc:15]: Not really to create an instance, but for managing the configuration of an instance.

File: docs/topics/mta-web-administrator-view.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+//![](/Tackle2/Views/AdminMenu.png)
+
+=== Credentials
+The credentials management module enables administrators to create and manage credentials to access private repositories.  It enables architects to assign credentials to applications without having to know their contents.
+//![](/Tackle2/Views/CredMain.png)
+
```
Comment (rromannissen) [inline docs/topics/mta-web-administrator-view.adoc:21]: The Credentials view doesn't allow to assign credentials to applications. That happens in the Application Inventory view.

File: docs/topics/mta-web-administrator-view.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+* *Type*
+** *Source Control*
+** *Username/Password*
+** *Source Private Key/Passphrase*
+** *Maven Settings File*
+** *Proxy*
+* *Type specific information*
+* *Created by*
+
+NOTE: Credential information entered, such as keys and passphrases, is hidden or displayed as [Encrypted].
```
Comment (rromannissen) [inline docs/topics/mta-web-administrator-view.adoc:35]: This is not true for Maven Settings files upon creation. When the file is uploaded, its contents are displayed in the text box for the user to make tweaks if desired. [Encrypted] only appears on edit for Maven Settings files.

File: docs/topics/mta-web-administrator-view.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+* *Git configuration*
+** *Consume insecure Git repositories*
+//![](/Tackle2/Views/SubvConfig.png)
+* *Subversion configuration*
+** *Consume insecure Subversion repositories*
+//![](/Tackle2/Views/MavenConfig.png)
+
+* *Maven configuration*
+** Manage repository size
+** *Force update of dependencies*
```
Comment (rromannissen) [inline docs/topics/mta-web-administrator-view.adoc:50]: I don't think this is an option anymore. @jortel please confirm.

File: docs/topics/mta-web-administrator-view.adoc (hunk 1 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-administrator-view.adoc]: I made the requested changes, please review.

File: docs/topics/mta-web-administrator-view.adoc (hunk 2 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-administrator-view.adoc:21]: I made the requested changes, please review.

File: docs/topics/mta-web-administrator-view.adoc (hunk 3 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-administrator-view.adoc]: I made the requested changes, please review.

File: docs/topics/mta-web-administrator-view.adoc [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+** *Source Private Key/Passphrase*
+** *Maven Settings File*
+** *Proxy*
+* *Type specific information*
+* *Created by*
+
+NOTE: Credential information entered, such as keys and passphrases, is hidden or displayed as [Encrypted].
+
+=== Repositories
+The repository management module configures the repositories used by {ProductShortName} {WebName} and has the following options:
```
[the remark this answers is not in the document]
Comment (CarmiWisemon) [reply docs/topics/mta-web-administrator-view.adoc]: I made the requested changes, please review.

File: docs/topics/mta-web-administrator-view.adoc (hunk 4 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-administrator-view.adoc:50]: @rromannissen waiting for a response.

File: docs/topics/mta-web-administrator-view.adoc [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+** *Source Control*
+** *Username/Password*
+** *Source Private Key/Passphrase*
+** *Maven Settings File*
+** *Proxy*
+* *Type specific information*
+* *Created by*
+
+NOTE: Credential information entered, such as keys and passphrases, is hidden or displayed as [Encrypted], except for Maven Settings files.  When Maven Setting files are created, the file is uploaded and its contents are displayed in the text box to enable the user to make changes if desired. [Encrypted] only appears on edit for Maven Settings files.
+
```
Comment (kpunwatk) [inline docs/topics/mta-web-administrator-view.adoc:36]: Please re-check and update this note.
 Now, The user can also see and change the contents of keys and passphrases
Screenshot from 2022-11-03 21-15-07

Comment (CarmiWisemon) [reply docs/topics/mta-web-administrator-view.adoc:36]: @rromannissen please see above. How should I edit this note based on this information?

Comment (CarmiWisemon) [reply docs/topics/mta-web-administrator-view.adoc:36]: @rromannissen Should I remove the note?

Comment (rromannissen) [reply docs/topics/mta-web-administrator-view.adoc:36]: I don't think this is accurate, the example here is for the SCM Private Key file, not for the Maven settings.xml file. I think this should remain as it is right now

File: docs/topics/mta-web-administrator-view.adoc [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+= User Interface views
+
+The {ProductName} ({ProductShortName}) {WebName} and has two user interfaces with different functionalities:
+
+* Administrator view
+* Developer view
+
+== Administrator view
+The administrator view is used by administrators of {ProductShortName} {WebName} to manage the configuration of an instance.
+//![](/Tackle2/Views/AdminMenu.png)
```
Comment (RichardHoch) [inline docs/topics/mta-web-administrator-view.adoc:16]: "If you are an administrator, you can use *Admninstrator* view to...

File: docs/topics/mta-web-administrator-view.adoc [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+=== Credentials
+The credentials management module enables administrators to create and manage credentials to access private repositories.
```
Comment (RichardHoch) [inline docs/topics/mta-web-administrator-view.adoc:19]: "You can use *Credentials* view to... "

File: docs/topics/mta-web-administrator-view.adoc [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+* *Description*
+* *Type*
+** *Source Control*
+** *Username/Password*
+** *Source Private Key/Passphrase*
+** *Maven Settings File*
+** *Proxy*
+* *Type specific information*
+* *Created by*
+
```
Comment (RichardHoch) [inline docs/topics/mta-web-administrator-view.adoc:34]: See the example at the end of https://redhat-documentation.github.io/supplementary-style-guide/#admonitions to see how to format notes. If you haven't read the entire article, do so in the near future.
https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

File: docs/topics/mta-web-administrator-view.adoc (hunk 6 above)
Comment (RichardHoch) [inline docs/topics/mta-web-administrator-view.adoc:36]: "Maven settings," not "Maven Settings"

File: docs/topics/mta-web-administrator-view.adoc (hunk 9 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-administrator-view.adoc:34]: Thank you. My bad.

File: docs/topics/mta-web-administrator-view.adoc (hunk 6 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-administrator-view.adoc:36]: Done

**Pull request merged**